test/e2e/util.go

```diff
@@ -177,6 +177,10 @@ func Logf(format string, a ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, "INFO: "+format+"\n", a...)
 }
 
+func Errorf(format string, a ...interface{}) {
+	fmt.Fprintf(GinkgoWriter, "EROR: "+format+"\n", a...)
```

Comment:
OK, I'm removing it.

(I never figured this out, but I wanted it to print in red; that would have been cool.)